modules/docs/llm/upgrade-guides/16.0-UPGRADE-GUIDE.md

```
@@ -236,6 +246,23 @@ import {DeleteButton} from '@workday/canvas-kit-react/button';
 - Error and caution states no longer display a status background color; only the standard background
   is shown.
 
+### Segmented Control
+
+**PR:** [#4048](https://github.com/Workday/canvas-kit/pull/4048)
+
+#### Visual Updates
+
+#### Visual Updates
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)